vue-napredno/src/views/user/ProfileView.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="pa-6 d-flex">
+    <v-col md="2">
+      <ValidationObserver v-slot="{ handleSubmit }" ref="updateUserForm">
```

Comment:
ValidationObserver i ValidationProvider registrirati globalno i imenovati ih s [kebab-casom](https://www.codingem.com/what-is-kebab-case/).